src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/Resources/views/Teams/Table/role.html.twig

```diff
@@ -4,39 +4,57 @@
-{% set iconAfter %}
-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&#xf0d7;
-{% endset %}
 {% if user is defined %}
     {% set url = path('team_change_role', {'memberId': user.id, 'newRole': 'stand-in' }) %}
 {% endif %}
 
-<label for="role-select"{% if labelClass is defined %} class={{ labelClass }}{% endif %}>{% if labelText is defined %}{{ labelText }}{% endif %}</label>
-<select id="role-select" class="teams__select"{% if url is defined %} data-url="{{ url }}"{% endif %}{% if isInvite is defined and isInvite %} disabled {% endif %}>
-    <option class="fontawesome" data-icon="fa-star" value="admin" {% if userRole == 'admin' %}selected="selected"{% endif %}>
-        &#xf005;{{ whitespaceBefore }}
-        {{ 'teams.management.table.role.admin'|trans }}
-        &nbsp;&nbsp;&nbsp;{{ iconAfter }}
-    </option>
-    <option class="fontawesome" data-icon="fa-universal-access" value="owner" {% if userRole == 'owner' %}selected="selected"{% endif %}>
-        &#xf29a;{{ whitespaceBefore }}
-        {{ 'teams.management.table.role.owner'|trans }}
-        &nbsp;&nbsp;&nbsp;{{ iconAfter }}
-    </option>
-    <option class="fontawesome" data-icon="fa-black-tie" value="manager" {% if userRole == 'manager' %}selected="selected"{% endif %}>
-        &#xf27e;{{ whitespaceBefore }}
-        {{ 'teams.management.table.role.manager'|trans }}
-        {{ iconAfter }}
-    </option>
-    <option class="fontawesome" data-icon="fa-user" value="member" {% if userRole == 'member' %}selected="selected"{% endif %}>
-        &#xf007;{{ whitespaceBefore }}
-        {{ 'teams.management.table.role.member'|trans }}
-        &nbsp;{{ iconAfter }}
-    </option>
-</select>
+<div tabindex="0" {% if url is defined %} data-url="{{ url }}"{% endif %}>
+            <span class="pull-left role__admin {% if userRole != 'admin' %}hidden{% endif %}">
+                <i class="fa fa-star fa-fw" aria-hidden="true"></i>
+                {{ 'teams.management.table.role.admin'|trans }}
+            </span>
+            <span class="pull-left role__owner {% if userRole != 'owner' %}hidden{% endif %}">
+                <i class="fa fa-universal-access fa-fw" aria-hidden="true"></i>
+                {{ 'teams.management.table.role.owner'|trans }}
+            </span>
+            <span class="pull-left role__manager {% if userRole != 'manager' %}hidden{% endif %}">
+                <i class="fa fa-black-tie fa-fw" aria-hidden="true"></i>
+                {{ 'teams.management.table.role.manager'|trans }}
+            </span>
+            <span class="pull-left role__member {% if userRole != 'member' %}hidden{% endif %}">
+                <i class="fa fa-user fa-fw" aria-hidden="true"></i>
+                {{ 'teams.management.table.role.member'|trans }}
+            </span>
+    <i class="fa fa-spinner fa-spin loading"></i>
+    <i class="fa fa-caret-down" aria-hidden="true"></i>
+    <ul>
+        <li class="teams__actionItem">
+            <a class="teams__changeMemberRole" role="admin">
```

Comment:
This is an a-tag without a href attribute.  That should never happen.  If this is a link, then it needs a href attribute.  If it isn't then it should likely be a button (not sure what the item should do when clicked, but if it's just to change member role without changing page, then a button is best.  This should likely be a button?

Also: the role attribute is an existing attribute, So you shouldn't use that, it should be a data-role attribute.  See https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles for more information.

Reply:
an href="#" has been included
role has been renamed to data-role

Reply:
I know adding a `href="#"` is a ubiquitous practice, however it's invalid HTML and therefore a bad practice.

If you turn it into a button, then you can use `initial: all` to get rid of all browser styling for the button 😉

Reply:
see: https://css-tricks.com/almanac/properties/a/all/

Reply:
I could not get the buttons styled easily, 
The other dropdowns use <a> tags too with an href. Restyling the <button> elements will take too much time for now. 

After asking  @quartje we have agreed to add the remainder  of the work as a technical debt ticket
- Keyboard accesibility
- Use buttons instead of a tags within the dropdown template.
see:
-https://www.pivotaltracker.com/story/show/182261042
- https://www.pivotaltracker.com/story/show/182261075